nixnet/_enums.py

```diff
@@ -2029,3 +2029,27 @@ class FrameType(enum.Enum):
     SPECIAL_DELAY = _cconsts.NX_FRAME_TYPE_SPECIAL_DELAY
     SPECIAL_LOG_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_LOG_TRIGGER
     SPECIAL_START_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_START_TRIGGER
+
+
+class Terminal(enum.Enum):
+    "Terminals to import/export from the XNET device"
+    FRONTPANEL_0 = _cconsts.NX_TERM_FRONT_PANEL0
```

Comment:
Should FRONTPANEL_0 and FRONTPANEL_1 be removed? They are duplicates of FRONT_PANEL_0 and FRONT_PANEL_1 below.